sample-code/spring-app/pom.xml

```diff
@@ -189,6 +188,30 @@
           <skipTests>${skipTests}</skipTests>
         </configuration>
       </plugin>
+      <!-- Skip release -->
```

Comment:
(Minor)

I don't really like how complex it is for disabling a module for release. Can you instead reduce it to just a _property_ like `<release.skip>true</release.skip>` in this file and default `false` in parent file(?)

If yes, then you would add the `<skip>${release.skip}</skip>` to parent pom.

Reply:
That would be great if the `maven-deploy-plugin` had a `skip` property...
Not all plugins have skip properties.